<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parkingClass</class>
 <widget class="QMainWindow" name="parkingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>parking</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_LoadSTL"/>
    <addaction name="action_LoadDXF"/>
    <addaction name="action_Load3DS"/>
    <addaction name="action_LoadIGES"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <addaction name="menuHello"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_LoadSTL">
   <property name="text">
    <string>Load STL</string>
   </property>
  </action>
  <action name="action_LoadDXF">
   <property name="text">
    <string>Load DXF</string>
   </property>
  </action>
  <action name="action_Load3DS">
   <property name="text">
    <string>Load 3DS</string>
   </property>
  </action>
  <action name="action_LoadIGES">
   <property name="text">
    <string>Load IGES</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
